client/core/core.go

```
@@ -1747,7 +1760,7 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 	}
 
 	initErr := func(s string, a ...interface{}) error {
-		_ = wallet.Lock() // just try, but don't confuse the user with an error
+		_ = walletLockWithTimeout(2*time.Second, wallet) // just try, but don't confuse the user with an error
```

The error is ignored anyway and the wallet is immediately disconnected...